### ISSUE TITLE
Inconsistency in slide-down animation

### ISSUE THREAD

**Issue by RomanovX** [PolymerElements/neon-animation#140]
The readme shows the slide up and down animations to be:

 * `slide-down-animation` Animates transform from `translateY(-100%)` to `none`
 * `slide-up-animation` Animates transform from `none` to `translateY(-100%)`
 * `slide-from-top-animation` Animates transform from `translateY(-100%)` to `none`
 * `slide-from-bottom-animation` Animates transform from `translateY(100%)` to `none`

However the code suggests otherwise:

`slide-down-animation`:
```
this._effect = new KeyframeEffect(node, [
        {'transform': 'translateY(0%)'},
        {'transform': 'translateY(100%)'}
      ], this.timingFromConfig(config));
```
This, while the others are coherent. In #82 this was supposedly fixed. If consistent with the other animations, the readme should be changed to:
 * `slide-down-animation` Animates transform from `none` to `translateY(100%)`

Possibly with a note that the animation pairs are:
 * `slide-from-bottom-animation` and `slide-down-animation`
 * `slide-from-top-animation` and `slide-up-animation`

for easy and quick referencing.

**Comment by RomanovX** [comment]
I have fixed this in #141

**Comment by samuelli** [comment]
Thanks for fixing this!
